### PR TITLE
Admin Damage Menu

### PR DESCRIPTION
**Pull request by Geevies**
* Admins have received a Damage Menu, to access this, check out the Fun tab.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/modules/mob/living/carbon/human/damage_menu.dm [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+		organs[capitalize_first_letters(name)] = !!H.internal_organs_by_name[name]
+
+	data["limbs"] = limbs
+	data["organs"] = organs
+
+	return data
+
+/datum/vueui_module/damage_menu/Topic(href, href_list)
+	if(!usr.client.holder)
+		return
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/carbon/human/damage_menu.dm:54]: This needs to check more than just the holder. R_ADMIN specifically.

File: code/modules/mob/living/carbon/human/damage_menu.dm [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+	var/mob/living/carbon/human/H = target_human.resolve()
+	if(!istype(H))
+		return
+
+	switch(href_list["target"])
+		if("limb")
+			var/obj/item/organ/external/L = H.organs_by_name[reverse_parse_zone(lowertext(href_list["name"]))]
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/carbon/human/damage_menu.dm:62]: I have to say, this is the ugliest shit I've ever seen

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/damage_menu.dm (hunk 2 above)
Comment (Geevies) [reply code/modules/mob/living/carbon/human/damage_menu.dm:62]: self insert

**Review by NonQueueingMatt — APPROVED** [review]

**Review by JohnWildkins — APPROVED** [review]

**Pull request merged**